src/status_im2/contexts/onboarding/enter_seed_phrase/view.cljs

```diff
@@ -0,0 +1,75 @@
+(ns status-im2.contexts.onboarding.enter-seed-phrase.view
+  (:require [quo2.core :as quo]
+            [quo.core :as quo1]
+            [quo2.foundations.colors :as colors]
+            [clojure.string :as string]
+            [status-im.ethereum.mnemonic :as mnemonic]
+            [utils.security.core :as security]
+            [utils.re-frame :as rf]
+            [reagent.core :as reagent]
+            [react-native.core :as rn]
+            [status-im2.contexts.onboarding.enter-seed-phrase.style :as style]
+            [status-im2.contexts.onboarding.common.background.view :as background]
+            [utils.i18n :as i18n]))
+
+(defn navigation-bar
+  []
+  [rn/view {:style style/navigation-bar}
+   [quo/page-nav
```

Comment:
Just curious, why we need overlapping view. from styles navigation bar has height 56, page-nav already has that
https://github.com/status-im/status-mobile/blob/a502da6ea43964f485e99e56c86a962da260a05a/src/quo2/components/navigation/page_nav.cljs#L228

Reply:
(I copied and pasted the code :) but I'll look into it